### PR TITLE
Dp 16111 multiselect dropdown

### PR DESCRIPTION
**Pull request by clairesunstudio**
Minor
Added
- (React) [MultiSelectDropDown] Added `MultiSelectDropDown` molecule. #787
- (React) [ButtonTag, Tags] Added `ButtonTag` atom under buttons and `Tags` molecule. #787

Changed
- (React) [InputCheckBox] Refactor InputCheckBox to be used for multiple selections. #787
  - Allow defaultValue to update the component state
  - Added required `value` prop to input checkbox element ([Breaking Change Note](https://github.com/massgov/mayflower/pull/787#issuecomment-543820137))
  - Added `onKeyDown` prop to allow passing arrow navigation between checkbox options
  - Added `tabIndex` prop to permit change of tab-index
  - Added `classes` prop to permit passing in classes to input wrapper div
- (React) [Input] Added classes props to Input to pass classNames into input wrapper div. #787

### CODE REVIEW

**Comment by clairesunstudio** [comment]
This is pr adds a required prop `value` to InputCheckBox for allowing multiple selection. This is a breaking change and requires the consumer of InputCheckBox in this mayflower version to add `value` prop. The context value of this input will be toggled between `value` and false.

**Comment by clairesunstudio** [comment]
> This is pr adds a required prop `value` to InputCheckBox for allowing multiple selection. This is a breaking change and requires the consumer of InputCheckBox in this mayflower version to add `value` prop. The context value of this input will be toggled between `value` and false.

File: react/src/components/atoms/forms/InputCheckBox/index.js [hunk 1]
```diff
@@ -88,14 +102,20 @@ const InputCheckBox = (props) => {
 InputCheckBox.propTypes = {
   /** Id of the input that the label is tied to and the value is associated with in the formContext. */
   id: PropTypes.string,
-  /** Default checked value. */
-  defaultValue: PropTypes.bool,
+  /** Value of the input that is associated with in the formContext. */
+  value: PropTypes.string,
```
Comment (clairesunstudio) [inline react/src/components/atoms/forms/InputCheckBox/index.js]:
```suggestion
  value: PropTypes.string.isRequired,
```

File: react/src/components/atoms/forms/InputCheckBox/index.js [hunk 2]
```diff
@@ -88,14 +102,20 @@ const InputCheckBox = (props) => {
 InputCheckBox.propTypes = {
   /** Id of the input that the label is tied to and the value is associated with in the formContext. */
   id: PropTypes.string,
-  /** Default checked value. */
-  defaultValue: PropTypes.bool,
+  /** Value of the input that is associated with in the formContext. */
```
Comment (clairesunstudio) [inline react/src/components/atoms/forms/InputCheckBox/index.js]:
```suggestion
  /** Value of the input that is associated with in the formContext. (required) */
```

**Review by ygannett — APPROVED** [review]
It looks good.

**Comment by ygannett** [comment]
ci/circleci: patternlab_test failure can be ignored since it's Google map content change.

**Pull request merged**